ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanelComponent.stories.tsx

```diff
@@ -89,19 +100,10 @@ const meta: Meta<typeof TimeSeriesChart.PanelComponent> = {
             {
               query: 'up{job="grafana",instance="demo.do.prometheus.io:3000"}',
               response: {
-                body: mockTimeSeriesResponseWithStableValue({
-                  metrics: [
-                    {
-                      metric: {
-                        __name__: 'up',
-                        instance: 'demo.do.prometheus.io:3000',
-                        job: 'grafana',
-                      },
-                      value: '1',
-                    },
-                  ],
+                body: mockTimeSeriesResponseWithManySeries({
```

Comment:
Changed this to a mock that shows a bunch of series, which is better for showing off the legends.